store/sqlstore/command_webhook_store.go

```diff
@@ -70,11 +69,10 @@ func (s SqlCommandWebhookStore) Get(id string) (*model.CommandWebhook, error) {
 
 	queryString, args, err := query.ToSql()
 	if err != nil {
-		return nil, model.NewAppError("SqlCommandWebhookStore.Get", "store.sql.build_query.app_error", nil, "id="+id+", err="+err.Error(), http.StatusInternalServerError)
+		return nil, errors.Wrap(err, "audits_tosql")
```

Comment:
The error message here makes no sense. this should be get_command_webhook_tosql or something like that, the same fro TryUse